### PR TITLE
Prop uri required

### PR DESCRIPTION
**Pull request by jgreben**
@ndushay I used the pattern that I set up in the `export-profile.test` which seems to work better. Also used http://jsonselector.com in order to find the correct selector for the propertyURI.

### CODE REVIEW

File: source/__tests__/integration/create-property.test.js [hunk 1]
```diff
@@ -43,23 +35,82 @@ describe('Create profile has properties for resource and templates for property'
...
+  })
+})
+
+describe('property URI is required', () => {
+  let profileFormSel = 'form[name="profileForm"]'
+  let exportButtonSel = 'a.btn.import-export'
+
+  // beforeEach(async () => {
+  //   await page.$eval(profileFormSel, e => e.reset(), {timeout: 1000})
+  // })
```
Comment (ndushay) [inline source/__tests__/integration/create-property.test.js]: should these commented out lines come out?  Or does it not matter?

Comment (ndushay) [reply source/__tests__/integration/create-property.test.js]: @jermnelson ^^   wanna decide?  Merge or not?

Comment (jgreben) [reply source/__tests__/integration/create-property.test.js]: I meant to remove those lines..

File: source/__tests__/integration/create-property.test.js [hunk 2]
```diff
@@ -43,23 +35,78 @@ describe('Create profile has properties for resource and templates for property'
...
+      description: "Profile description",
+      author: "Me",
+      title: "My profile",
+      resourceId: "my:resource",
+      resourceURI: "http://www.example.com"
+    })
+
+    await page.click(exportButtonSel)
+    let alertBoxSel = '#alertBox > div.modal-dialog > div.modal-content > div.modal-body > p#alert_text'
+    await expect_value_in_sel_text(alertBoxSel, 'Parts of the form are invalid')
```
Comment (ndushay) [inline source/__tests__/integration/create-property.test.js]: FYI, in my branch, which doesn't even include this, I just got this:

```
server/whichrt called for uri other than ontology http://www.example.com
 FAIL  source/__tests__/integration/create-resource.test.js
  ● Create profile resource template requirements › resource template form fields › requires Resource URI

    expect(received).toBe(expected) // Object.is equality

    Expected: "Parts of the form are invalid"
    Received: "my:resource must have at least one property template"

      86 | async function expect_value_in_sel_textContent(sel, value) {
      87 |   const sel_text = await page.$eval(sel, e => e.textContent)
    > 88 |   expect(sel_text).toBe(value)
         |                    ^
      89 | }
      90 | 
```

which is probably the error message we want in this case, no?

Comment (jgreben) [reply source/__tests__/integration/create-property.test.js]: @ndushay this was a WIP. Sees like there was an issue with the form fills not happening in the correct order and `waitUntil: 'networkidle2'` was the thing that fixed it.

**Pull request merged**